app/src/main/java/com/ucmobiledevelopment/freeways/ReportIncidentActivity.kt

```diff
@@ -18,21 +17,22 @@ class ReportIncidentActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            FreeWaysTheme() {
-                // A surface container using the 'background' color from the theme
+            FreeWaysTheme {
+                //A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                  //  Greeting("Android")
+
```

Comment:
Deleted the comment it is not needed for the code to function. https://kotlinlang.org/docs/coding-conventions.html#documentation-comments